src/inbox/mod_inbox_utils.erl

```diff
@@ -16,21 +16,38 @@
 
 %%%%%%%%%%%%%%%%%%%
 %% DB Operations shared by mod_inbox_one2one and mod_inbox_muclight
--export([reset_unread_count/3,
+-export([maybe_reset_unread_count/4,
+         reset_unread_count/3,
```

Comment:
Is this export still used?

Reply:
Good catch.

Reply:
Actually, nope. Removing this export makes a ton of tests fail as well.